meteor/client/ui/Shelf/ColoredBoxPanel.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react'
+import * as _ from 'underscore'
+import {
+	DashboardLayoutColoredBox,
+	RundownLayoutBase,
+	RundownLayoutColoredBox,
+} from '../../../lib/collections/RundownLayouts'
+import { MeteorReactComponent } from '../../lib/MeteorReactComponent'
+import { RundownPlaylist } from '../../../lib/collections/RundownPlaylists'
+import { dashboardElementPosition } from './DashboardPanel'
+import { RundownLayoutsAPI } from '../../../lib/api/rundownLayouts'
+import { Translated } from '../../lib/ReactMeteorData/ReactMeteorData'
+import { withTranslation } from 'react-i18next'
+
+interface IColoredBoxPanelProps {
+	visible?: boolean
+	layout: RundownLayoutBase
+	panel: RundownLayoutColoredBox
+	playlist: RundownPlaylist
+}
+
+interface IState {}
+
+interface IColoredBoxPanelTrackedProps {
+	name?: string
```

Comment:
Why is this in a TrackedProps interface? This component doesn't use a tracker.

Reply:
I've removed the tracked props https://github.com/tv2/tv-automation-server-core/commit/cbf2f5260a259087d4d4c0838c2414f0081fd0e6